rest_framework/urlpatterns.py

```diff
@@ -1,5 +1,5 @@
-from django.conf.urls import include, url
-from django.urls import URLResolver, path, register_converter
+from django.conf.urls import include
```

Comment:
May as well update the `include` import (to `from django.urls`) as well

Reply:
Yeah, I have updated.